openmdao/docs/features/core_features/working_with_derivatives/approximating_totals.rst

```diff
@@ -53,12 +55,23 @@ You can also complex step your model or group, though there are some important r
   satisfy this requirement. Take care with functions like :code:`abs`, which effectively squelches
   the complex part of the argument.
 
-**Solvers like Newton that require gradients are not supported:**
-  Complex stepping a model causes it to run with complex inputs. When there is a nonlinear solver at
-  some level, the solver must be able to converge. Some solvers such as :code:`NonlinearBlockGS` can
-  handle this. However, the Newton solver must linearize and initiate a gradient solve about a
-  complex point. This is not possible to do at present (though we are working on some ideas to make
-  this work.)
+**If you complex step around a solver that requires gradients, the solver must not get its gradients from complex step:**
+  When you complex step around a nonlinear solver that requires gradients (like Newton), the
+  nonlinear solver must solve a complex linear system rather than a real one. Most of OpenMDAO's linear solvers
+  (with the exception of `PetscKSP`) support the solution of such a system.  However, when you linearize the submodel
+  to determine derivatives around a complex point, the application of complex step loses some of its robust properties
+  when compared to real-valued finite difference (in particular, you get subtractive cancelation which causes
+  increased inaccuracy for smaller stepsizes.)
```

Comment:
The wording here is unclear.... and I think we can work around this problem and allow complex for the partials if needed. Lets talk